### PR TITLE
Bundle A. Final CSS + HTML changes to the Landing page

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?
In the context of the bundle's A/B test, this PR add all the CSS regarding the first design.

## Changes
* Created the file structure for design A and B.
* Finished all the CSS related to design A.

### Desired design

desired design.png)

### Final screenshot

final-a

### CODE REVIEW

File: frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                                @icon
+                            }
+                            </div><div class="bundle-offering-a__subscribe__offer__list__item__content">
+                                <h1>Print</h1>
+                                <p>Pick the package you want: Saturday, Sunday, Weekly, Weekend or Everyday.  Save up to 36% on the Guardian and Observer newspapers</p>
+                            </div>
+                        </li>
+                    </ul>
+                    @fragments.bundle.bundleButton("See options","#","", chevron = true)
+                    <div class="bundle-offering-a__subscribe__offer__print-options">
```
Comment (SiAdcock) [inline frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html]: Possibly redundant `div`? We could move the class to the `ul`

File: frontend/assets/images/inline-svgs/raw/bundle-adfree.svg [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```
Comment (SiAdcock) [inline frontend/assets/images/inline-svgs/raw/bundle-adfree.svg]: I think we can strip the top 3 lines from this SVG

File: frontend/assets/images/inline-svgs/raw/bundle-adfree.svg [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (SiAdcock) [inline frontend/assets/images/inline-svgs/raw/bundle-adfree.svg]: I don't think we need `version`, `id`, `xmlns:xlink`, `x` or `y` attributes

File: frontend/assets/stylesheets/components/_bundle-offerings-a.scss [hunk 4]
```diff
@@ -68,66 +55,133 @@
...
+    }
+}
+.bundle-offering-a__subscribe__offer__print-options__option {
+    background-color: #ffffff;
+    border-radius: 10px;
+
+    height:52px;
+    margin: $gs-baseline/2 10px 0px;
+
+    padding: 8px $gs-gutter/2 0px;
```
Comment (SiAdcock) [inline frontend/assets/stylesheets/components/_bundle-offerings-a.scss]: Zeros don't need `px` units

Let's kill these black lines

File: frontend/assets/images/inline-svgs/raw/bundle-adfree.svg [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="28px" height="30px" viewBox="0 0 28 30" enable-background="new 0 0 28 30" xml:space="preserve">
+<g>
+	<path fill="#333333" d="M8,10c5.523,0,10-4.477,10-10c0,5.523,4.477,10,10,10c-5.523,0-10,4.478-10,10C18,14.478,13.523,10,8,10"/>
```
Comment (SiAdcock) [inline frontend/assets/images/inline-svgs/raw/bundle-adfree.svg]: _Maybe_ these `fill` values can be moved to the CSS to make these SVGs more reusable?

File: frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html (hunk 1 above)
Comment (svillafe) [inline frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html]: That `div` has the purpose of wrapping the entire section of print options (`ul` and the `button`). In that way, in the future we will be able to show / hide that entire section.

**Comment by svillafe** [comment]
Tested in browserStack with IE 11 and everything is working.

**Pull request merged**